cash-register/js/ItemList.js

```diff
@@ -0,0 +1,22 @@
+// object containing all items available for sale
+var itemList = {
+  milk: new Item('milk', 2.00),
+    eggs: new Item('eggs', 3.00),
+    chocolate: new Item('chocolate', 2.50),
+    kittyFood: new Item('kitty food', 20.00)
+};
+
+// add new items to itemList
+
+var addNewItem = function(name, price) {
+  itemList[name] = new Item(name,price);
+};
+
+addNewItem('candy',4);
+addNewItem('popcorn',5);
+
+// // to view/log itemList
```

Comment:
@meowbot24 Style - If you don't want code then just delete it :) Git has version control to go back in history if you want it back. :)